elide-datastore/elide-datastore-hibernate5/pom.xml

```diff
@@ -37,6 +37,10 @@
         <url>https://github.com/yahoo/elide.git</url>
         <tag>HEAD</tag>
     </scm>
+    <properties>
+        <excludeTags>emptyOnAttributeCollection</excludeTags>
```

Comment:
Looks like the fix for querying attribute collection on EMPTY operator is fixed in version `5.4.11`. We are using `5.4.2.Final`. 
One option is we use `where size(book.awards) = 0` as suggested in below link. 
Or we could continue using Empty operator and change the version to 5.4.11 when it gets released.

https://hibernate.atlassian.net/browse/HHH-6686?attachmentOrder=asc&oldIssueView=true
http://hibernate.org/orm/releases/5.4/

Reply:
We can update Hibernate version on this branch as well.

Reply:
It is not released yet http://hibernate.org/orm/releases/5.4/